### PR TITLE
Rhmap 9474 lazy load

### PR DESCRIPTION
**Pull request by witmicko**
# Reasoning 
Part of [RHMAP-9474](https://issues.jboss.org/browse/RHMAP-9474), lazy loading AeroGear module saves ~12MB of memory if push is not used.

@wei-lee

### CODE REVIEW

File: lib/push.js [hunk 1]
```diff
@@ -19,16 +19,24 @@ module.exports = function (cfg) {
     headers: headers
   };
 
-  var sender = AeroGear.Sender(settings);
 
   // $fh.push
+  var sender;
   return function push(message, options, callback) {
+    if (!aeroGearLoaded) {
+      AeroGear = require("unifiedpush-node-sender");
```
Comment (fh-qube) [inline lib/push.js]: MAJOR Add the "var" keyword to this declaration of "AeroGear". [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2703)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by wei-lee** [comment]
LGTM

**Pull request merged**